apps/admin-gui/src/app/shared/components/create-service-member-dialog/create-service-member-dialog.component.ts

```diff
@@ -151,11 +176,39 @@ export class CreateServiceMemberDialogComponent implements OnInit {
     }, () => this.processing = false);
   }
 
+  parseNamespaceRules(){
+    for (const rule of this.namespaceRules) {
+      this.namespaceOptions.push(rule.namespaceName);
+
+      const fieldTypes =  {login: 'disabled'};
+      this.parseAttributes(fieldTypes, rule.requiredAttributes, 'required');
+      this.parseAttributes(fieldTypes, rule.requiredAttributes, 'optional');
```

Comment:
Use `optionalAttributes` with `optional` type.

Reply:
Thanks, fixed. For the error, I suspect it is caused by missing password manager implementations for specific namespaces in development environment.